packages/templates/nextjs-marketing-demo/src/components/layout/Footer.tsx

```diff
@@ -0,0 +1,18 @@
+'use client';
```

Comment:
Is there a reason this and the header need to be client components?

Reply:
Definitely not needed in this one. It snuck in when I copy/pasted the Header component to create the Footer and I overlooked this line